### DISCUSSION TITLE
spring-boot-maven-plugin manager

### DISCUSSION THREAD

**Discussion by c0d1ngm0nk3y** [renovatebot/renovate#31422] [Suggest an Idea]
### Tell us more.

Cloud Native Buildpacks can be used in the [spring-boot-maven-plugin](https://docs.spring.io/spring-boot/maven-plugin) the configuration ([example](https://docs.spring.io/spring-boot/maven-plugin/build-image.html#build-image.examples.buildpacks)) contains references to docker images.

Support for this configuration would be a nice addition to Renovate, also the implementation could almost entirely rely on the existing docker datasource, packages extraction is a trivial task in this case.

**Comment by nicolasbender** [comment]
Proposed implementation: https://github.com/renovatebot/renovate/pull/31425
